docs/docs/sourcing-from-ghost.md

```diff
@@ -91,13 +101,12 @@ exports.createPages = ({ graphql, actions }) => {
             },
           })
         })
-
         return resolve()
       })
     )
   })
 
-  return Promise.all(createPosts)
+  return Promise.all([createPosts])
```

Comment:
Why do we need this Promise.all? Can't it just return createPosts?

Reply:
@sidharthachatterjee We talked about that on Slack: It aligns better with the tutorial (on the docs) and for other queries this will be necessary (if followed like in the docs).

If you agree please Approve and merge.